<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fight_game</class>
 <widget class="QWidget" name="fight_game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="red_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>红队</string>
   </property>
  </widget>
  <widget class="QLabel" name="blue_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>蓝队</string>
   </property>
  </widget>
  <widget class="QLabel" name="battle_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>battle</string>
   </property>
  </widget>
  <widget class="QLabel" name="red_pokemon_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>171</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>red_pokemon</string>
   </property>
  </widget>
  <widget class="QLabel" name="blue_pokemon_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>161</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>blue_pokemon</string>
   </property>
  </widget>
  <widget class="QLabel" name="vs">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="red_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="blue_name">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="log">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>216</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
